packages/blade/package.json

```diff
@@ -276,15 +276,15 @@
       "name": "Import Button only",
       "path": "./build/components/index.web.js",
       "import": "{ Button }",
-      "limit": "16.6 kb",
+      "limit": "17 kb",
       "running": false
     },
     {
       "name": "Import Text only",
       "path": "./build/components/index.web.js",
       "import": "{ Text }",
-      "limit": "11.9 kb",
+      "limit": "13 kb",
```

Comment:
Text imports a few Box (styled props) things which impacts the bundle size by 145B